### ISSUE TITLE
Too many notifications

### ISSUE THREAD

**Issue by ksurya** [brianpetro/obsidian-smart-connections#641]
Seriously! 

It's too distracting.

Can we please have an option to mute _all_ notifications?

**Comment by bbecausereasonss** [comment]
I agree, would be nice to at least mute some of them like the API one which is useless mostly.

**Comment by brianpetro** [comment]
The only notification that should not be mutable is the embedding processing notification, and that's because this process can cause performance issues. So with this notification you are able to pause that process. 

If there are any notifications that do not have the mute button, you can screenshot and share them here and I can make sure that the mute button gets added. 🌴

**Comment by AW2307** [comment]
Regarding the embedding notifications: 

Why not give users an option here as well? I have not seen performance implications and would also prefer to mute all notifications or disable them with a global switch.

**Comment by brianpetro** [comment]
@bbecausereasonss what is the API notification you're referring to? Can you send a screenshot next time it occurs? 🌴

**Comment by RyanArmiger** [comment]
This one seems to pop up frequently. And I'm on v2.1.80 
![Screenshot 2024-07-02 at 09 57 02](https://github.com/brianpetro/obsidian-smart-connections/assets/25403543/60c6a483-9385-4bd7-bb2c-7cbb4dfccb1e)

**Comment by JamieAUS** [comment]
![Screenshot 2024-07-07 at 4 02 23 PM](https://github.com/brianpetro/obsidian-smart-connections/assets/42209421/c878b6e4-6447-4c07-8823-a55498459e94)
